### PR TITLE
Made Footer responsive

### PR DESCRIPTION
**Pull request by Saipradyumnagoud**
# Issue Title: 
Made Footer responsive
*closes* #226 

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Saipradyumnagoud** [comment]
@singodiyashubham87 You can review it now

File: package-lock.json
Comment (singodiyashubham87) [inline package-lock.json:1]: You don't need to change this file.

**Comment by Saipradyumnagoud** [comment]
> @Saipradyumnagoud I told you to read the discussion, you didn't have to create new pages for all the links & redirect users on click, You were just supposed to create a popup showing the actual content, like when clicked on policies, the popup will show policies in the pop up and so on...

Yes i will work on

**Comment by singodiyashubham87** [comment]
@Saipradyumnagoud We also don't want ourTeam.jsx & the changes in the package-lock.json file.

**Comment by Saipradyumnagoud** [comment]
> @Saipradyumnagoud We also don't want ourTeam.jsx & the changes in the package-lock.json file.

yes I will I want 1-2 days time

**Comment by Saipradyumnagoud** [comment]
@singodiyashubham87 i will create a new pull request

**Pull request closed without merging**